com.reprezen.swagedit/src/com/reprezen/swagedit/validation/Validator.java

```diff
@@ -118,6 +123,29 @@ public Validator() {
         return errors;
     }
 
+    protected Set<SwaggerError> checkMissingItemsKeyInArrayType(Model model) {
+        Set<SwaggerError> errors = new HashSet<>();
+        if (model != null && model.getRoot() != null) {
+            AbstractNode root = model.getRoot();
+            AbstractNode definitions = root.get("definitions");
+
+            if (definitions != null) {
+                for (AbstractNode n : definitions.elements()) {
+                    if (n.isObject() && n.get("type") instanceof ValueNode) {
+                        ValueNode typeValue = n.get("type").asValue();
+                        if ("array".equalsIgnoreCase(typeValue.getValue().toString())) {
+                            if (n.get("items") == null) {
+                                errors.add(new SwaggerError(n.getStart().getLine() + 1, IMarker.SEVERITY_ERROR,
+                                        Messages.error_array_missing_items));
```

Comment:
Straightforward check. 
I suggest rename `n` to `node`, otherwise it's hard to read. 
We can also extract a constant for the `"type"`, or create a method to retrieve it - `"type"` is used at least 10 times in SwagEdit code.